sdk/src/azure/core/az_mqtt5_topic_parser.c

```diff
@@ -185,11 +192,34 @@ AZ_NODISCARD az_result _az_mqtt5_topic_parser_extract_tokens_from_topic(
     az_span client_id,
     az_span service_id,
     az_span executor_id,
+    az_span sender_id,
     az_span* extracted_client_id,
     az_span* extracted_service_id,
     az_span* extracted_executor_id,
-    az_span* extracted_command_name)
+    az_span* extracted_sender_id,
+    az_span* extracted_name)
 {
+  if (extracted_client_id)
+  {
+    *extracted_client_id = AZ_SPAN_EMPTY;
```

Comment:
Won't this prevent you from copying into this span?

Reply:
It won't, these are set to AZ_SPAN_EMPTY by value